include/RED4ext/Span.hpp

```diff
@@ -9,12 +9,26 @@ namespace RED4ext
 template<typename T>
 struct Span
 {
+    Span() = default;
```

Comment:
Oh there was default ctor already, I just didn't see it. There's no need for another one.